### PR TITLE
R SFN tutorials

### PR DESCRIPTION
**Pull request by jasonge27**
Fixing inconsistent paths and typos in current R tutorials. Adding R tutorial for SFN

### CODE REVIEW

File: R/inst/tutorials/00-helloworld/README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ printing out 'Metaflow says: Hi!' to the terminal.**
 - Basics of Metaflow.
 - Step decorator.
 
+
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/00-helloworld```
+1. ```cd tutorials/00-helloworld```
```
Comment (savingoyal) [inline R/inst/tutorials/00-helloworld/README.md:12]: Can you also add instructions for RStudio? Here and every other tutorial.

File: R/inst/tutorials/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Tutorials for Metaflow R
+
+This set of tutorials provides a hands-on introduction to Metaflow. The basic concepts are introduced in practice, and you can find out more details about the functionality showcased in these tutorials in Basics of Metaflow and the following sections.
```
Comment (savingoyal) [inline R/inst/tutorials/README.md:3]: Can you a link to `Basics of Metaflow`?

**Review by savingoyal — CHANGES_REQUESTED** [review]

File: R/inst/tutorials/00-helloworld/README.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ printing out 'Metaflow says: Hi!' to the terminal.**
...
 2. ```Rscript helloworld.R show```
-3. ```Rscript helloworld.R run```
\ No newline at end of file
+3. ```Rscript helloworld.R run```
+
+In RStudio, you can run the script by directly run `source("helloworld.R")`.
```
Comment (savingoyal) [inline R/inst/tutorials/00-helloworld/README.md:16]: minor nit - _If you are using RStudio, you can run this script by directly executing `source("helloworld.R")`._

File: R/inst/tutorials/01-playlist/README.md [hunk 4]
```diff
@@ -11,8 +11,10 @@ in an R Markdown Notebook.**
...
-1. ```cd metaflow-tutorials/R/01-playlist/```
+1. ```cd tutorials/01-playlist/```
 2. ```Rscript playlist.R show```
 3. ```Rscript playlist.R run```
 4. ```Rscript playlist.R run --genre comedy```
-5. Open ```playlist.Rmd``` in RStudio.
\ No newline at end of file
+5. Open ```playlist.Rmd``` in RStudio.
+
+RStudio instructions. You can replace the last `run()` in `playlist.R` with `run(genre="comedy")`, and run the episode by `source("playlist.R")`.
```
Comment (savingoyal) [inline R/inst/tutorials/01-playlist/README.md:20]: minor nit - _If you are using RStudio, you can replace the `run()` in last line in `playlist.R` with `run(genre="comedy")`, and run the episode by executing `source("playlist.R")`._

File: R/inst/tutorials/01-playlist/README.md [hunk 5]
```diff
@@ -11,8 +11,10 @@ in an R Markdown Notebook.**
...
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/01-playlist/```
+1. ```cd tutorials/01-playlist/```
 2. ```Rscript playlist.R show```
 3. ```Rscript playlist.R run```
 4. ```Rscript playlist.R run --genre comedy```
-5. Open ```playlist.Rmd``` in RStudio.
\ No newline at end of file
+5. Open ```playlist.Rmd``` in RStudio.
```
Comment (savingoyal) [inline R/inst/tutorials/01-playlist/README.md:18]: Do I need `RStudio` here? Do I have no other way here? Maybe we can restructure the instructions as
#### To play this episode
Execute the flow -
##### on terminal
1. blah blah
2. blah blah
##### on RStudio
1. blah blah
2. blah blah

Inspect the flow -
##### on terminal
1. blah blah
2. blah blah
##### on RStudio
1. blah blah
2. blah blah

File: R/inst/tutorials/02-statistics/README.md [hunk 6]
```diff
@@ -17,7 +17,9 @@ $export METAFLOW_DEFAULT_METADATA=local
 ```
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/02-statistics```
+1. ```cd tutorials/02-statistics```
 2. ```Rscript stats.R show```
 3. ```Rscript stats.R run```
 4. Open ```02-statistics/stats.Rmd``` in RStudio
+
+In RStudio, you can run the episode by `source("stats.R")`.
```
Comment (savingoyal) [inline R/inst/tutorials/02-statistics/README.md:25]: Same comment as in L:16 in R/inst/tutorials/00-helloworld/README.md

File: R/inst/tutorials/03-playlist-redux/README.md [hunk 7]
```diff
@@ -14,9 +14,11 @@ $export METAFLOW_DEFAULT_METADATA=local
...
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R```
-2. ```Rscript 03-playlist-redux/playlist.R show```
-3. ```Rscript 03-playlist-redux/playlist.R run```
+1. ```cd tutorials/03-playlist-redux```
+2. ```Rscript playlist.R show```
+3. ```Rscript playlist.R run```
+
+In RStudio, you can run the script by directly run `source("playlist.R")`.
```
Comment (savingoyal) [inline R/inst/tutorials/03-playlist-redux/README.md:21]: Same comment as in L:16 in R/inst/tutorials/00-helloworld/README.md

File: R/inst/tutorials/04-helloaws/README.md [hunk 8]
```diff
@@ -15,10 +15,12 @@ use the client to access information about any flow from anywhere.**
 1. Configure your sandbox: https://docs.metaflow.org/metaflow-on-aws/metaflow-sandbox
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R```
-2. ```Rscript 04-helloaws/helloaws.R run```
+1. ```cd tutorials/04-helloaws```
+2. ```Rscript helloaws.R run```
 3. Open ```helloaws.Rmd' in your local RStudio
```
Comment (savingoyal) [inline R/inst/tutorials/04-helloaws/README.md:20]: Do we need `RStudio` here?

File: R/inst/tutorials/04-helloaws/README.md [hunk 9]
```diff
@@ -15,10 +15,12 @@ use the client to access information about any flow from anywhere.**
 1. Configure your sandbox: https://docs.metaflow.org/metaflow-on-aws/metaflow-sandbox
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R```
-2. ```Rscript 04-helloaws/helloaws.R run```
+1. ```cd tutorials/04-helloaws```
+2. ```Rscript helloaws.R run```
 3. Open ```helloaws.Rmd' in your local RStudio 
 
+In RStudio, you can run the script by `source("helloaws.R")`.
```
Comment (savingoyal) [inline R/inst/tutorials/04-helloaws/README.md:22]: Same comment as in L:16 in R/inst/tutorials/00-helloworld/README.md

File: R/inst/tutorials/05-statistics-redux/README.md [hunk 10]
```diff
@@ -20,7 +20,13 @@ because the data is being stored in AWS S3.**
 1. Configure your sandbox: https://docs.metaflow.org/metaflow-on-aws/metaflow-sandbox
 
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/02-statistics/```
+1. ```cd tutorials/02-statistics/```
 2. ```Rscript stats.R --package-suffixes=.R,.csv run --with batch --max-workers 4```
 3. Open ```02-statistics/stats.Rmd``` in your RStudio and re-run the cells. You can acccess
 the artifacts stored in AWS S3 from your local RStudio session. 
+
+In RStudio, you can replace the last line `run()` with
```
Comment (savingoyal) [inline R/inst/tutorials/05-statistics-redux/README.md:28]: Same comment as L:20 in R/inst/tutorials/01-playlist/README.md

File: R/inst/tutorials/07-autopilot/autopilot.Rmd [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+```{r}
+suppressPackageStartupMessages(library(metaflow))
+message("Current metadata provider: ", metaflow::get_metadata())
+```
+
+## Plot a timeline view of a scheduled run of MovieStatsFlow
+When you triggered your flow on AWS Step Functions using `step-functions trigger`, you would have seen an output similar to - 
+```{bash}
+...
+Workflow MovieStatsFlow triggered on AWS Step Functions (run-id sfn-b1bd0ec2-e29b-4818-90ad-507aa600dbf0).
```
Comment (savingoyal) [inline R/inst/tutorials/07-autopilot/autopilot.Rmd:18]: replace sfn-b1bd0ec2-e29b-4818-90ad-507aa600dbf0 with sfn-dolor-sit-amet here and in L:24

**Review by savingoyal — COMMENTED** [review]
Looks great! Some minor comments re: polish.

File: R/inst/tutorials/00-helloworld/README.md (hunk 3 above)
Comment (jasonge27) [reply R/inst/tutorials/00-helloworld/README.md:16]: updated according to your suggestion

File: R/inst/tutorials/01-playlist/README.md (hunk 4 above)
Comment (jasonge27) [reply R/inst/tutorials/01-playlist/README.md:20]: updated according to your suggestion

File: R/inst/tutorials/01-playlist/README.md (hunk 5 above)
Comment (jasonge27) [reply R/inst/tutorials/01-playlist/README.md:18]: R Markdown is built into RStudio https://rmarkdown.rstudio.com/ so for the best experiences I believe R Studio should be the recommended way to open it.

Good idea on restructuring the instructions! Updating it now.

File: R/inst/tutorials/04-helloaws/README.md (hunk 8 above)
Comment (jasonge27) [reply R/inst/tutorials/04-helloaws/README.md:20]: Yes RStudio is mostly for running the R markdown files

**Pull request merged**